src/main/java/randoop/util/Report.java

```diff
@@ -112,18 +114,18 @@ private void generateCSV(){
             PrintWriter writer = new PrintWriter(new File("methods_report.csv"));
```

Comment:
I think a refactor could be applied here (lines 112-128). So a method extraction might be performed.
The same applies for lines 138-160.